webapp/src/editortoolbar.tsx

```diff
@@ -208,6 +209,39 @@ export class EditorToolbar extends data.Component<ISettingsProps, EditorToolbarS
         this.props.parent.pairAsync();
     }
 
+    protected onCannotPairClick = async () => {
+        pxt.tickEvent("editortools.pairunsupported", undefined, { interactiveConsent: true });
+        const reasonUnsupported = await pxt.usb.getReasonUnavailable();
+        let modalBody: string;
+        switch (reasonUnsupported) {
+            case "security":
+                modalBody = lf("WebUSB appears to have been disabled by browser policy; check your browser's policy page at chrome://policy for more info.");
```

Comment:
Opinions on phrasing here would be helpful; my guess is that the number of users who are actively applying policies (vs having a policy applied by a school district) is... hand countable, so maybe this could just be phrased as "Check with your technical administrators if WebUSB can be enabled" or something of the sort

For reference, links to `chrome://whatever` from sites (anchor tags, window.open, etc) are not allowed, user has to type it in (paste it in) themselves. Also, `chrome://policy` does work in edge as well (redirects to `edge://policy`)

Reply:
<img width="208" alt="image" src="https://user-images.githubusercontent.com/5615930/236268996-e53ab0cd-daf0-4efd-883d-9f3a384c8ee1.png">
<img width="604" alt="image" src="https://user-images.githubusercontent.com/5615930/236269068-258c67d2-78b7-4993-b25d-5158ad0b7a02.png">

Updated re comments~